Cargo.toml

```diff
@@ -22,7 +22,7 @@ log4rs = "0.8"
 log4rs-syslog = "3.0"
 protobuf = "2"
 raft = "0.3.1"
-sawtooth_sdk = { git = "https://github.com/hyperledger/sawtooth-core.git" }
+sawtooth-sdk = "^0.1"
```

Comment:
How about extern crate declaration and use statements?
I guess build might fail without that change.

Reply:
If you mean due to the crate's coordinate change (`"sawtooth_sdk" -> "sawtooth-sdk"), those don't need to change.  Rust automatically translates those the dashes to the underscores.

(See https://github.com/rust-lang/rust/issues/23533 for details)

Reply:
Thanks for the link. It clarifies my question.